nano/node/rpc.cpp

```diff
@@ -2954,6 +2959,92 @@ void nano::rpc_handler::send ()
 	}
 }
 
+void nano::rpc_handler::sign ()
+{
+	// Retrieving hash
+	nano::block_hash hash (0);
+	boost::optional<std::string> hash_text (request.get_optional<std::string> ("hash"));
+	if (hash_text.is_initialized ())
+	{
+		hash = hash_impl ();
+	}
+	// Retrieving block
+	std::shared_ptr<nano::block> block;
+	boost::optional<std::string> block_text (request.get_optional<std::string> ("block"));
+	if (!ec && block_text.is_initialized ())
+	{
+		block = block_impl ();
```

Comment:
Right now you have to pass in a value for `signature` for the block, even though it will overwrite it.  Maybe better to supply our own.
```suggestion
		block = block_impl (true);
```

Reply:
Commited